neurodamus/ngv.py

```diff
@@ -303,17 +351,20 @@ def ustate_event_handler2(syn_gid):
                 if syn_gid is None:
                     continue
 
-            glut_obj = glut_list[int(syn_params.astrocyte_section_id)]
+            glut_idx = int(syn_params.astrocyte_section_id)
+            # check that we do not connect to the GlutSynapseSoma
```

Comment:
I'm not sure how this checks that it's not connected to `GlutSynapseSoma` (GlutSynapseSoma is not mentioned anywhere else in the file, I think) is it an invariant that it's at the end?
could also add the message to the assert

Reply:
eheheh, this was a todo/check for me. It is superfluous now